src/main/java/com/khadri/jdbc/prepare/data/driver/JdbcDataDriver.java

```diff
@@ -19,24 +19,20 @@
 public class JdbcDataDriver {
 	private Scanner scanner;
 
-	private EmployeeDao empDao;;
-
+	private EmployeeDao empDao;
 	private CustomerDao custDao;
-
 	private MovieDao movieDao;
 	private MobileDao mobileDao;
 	private SuperMarketDao superMarketDao;
```

Comment:
why you need superMarketDao in the Driver class? move to processor.

Reply:
SIR MORNING YOU ARE COMMENT IN MOVIE MODULE  AND TELL ALSO CHANGE EVERYONE